# Message that contains intermediate results 
# and diagnostics of the (predictive) planner.

std_msgs/Header header

# The planned trajectory (or if multiple plans exist, all of them)
teb_local_planner/TrajectoryMsg[] trajectories

# Index of the trajectory in 'trajectories' that is selected currently
uint16 selected_trajectory_idx

# List of active obstacles
costmap_converter/ObstacleArrayMsg obstacles_msg

# Total computation time
float32 computation_time

# start_pose id
string start_pose

# dt ref
float32 dt_ref
